### PR TITLE
Remove incorrect TMI from PFS description

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: docs/Catalog/Preview.md [hunk 1]
```diff
@@ -54,16 +54,15 @@ currently supported.
...
-File Server). It is at your discretion which JS library (or libraries)
-you wish to import in your HTML file.
-
-To enable "permissive" visualizations, check the `Enable permissive HTML
-rendering` checkbox in [Bucket settings](Admin.md#buckets). Please note
-that rendering Javascript-enabled HTML files in your S3 bucket or Quilt package
-creates a short-lived **publicly available** session for the lifetime of the
-session, although all session traffic _remains encrypted_.
+The Quilt Catalog supports HTML and JavaScript in preview via iframes. By default,
+preview iframes do not have access to S3.
```
Comment (dimaryaz) [inline docs/Catalog/Preview.md:58]: > By default, preview iframes do not have access to S3.

It took me a while to understand this sentence. My first thought was, "So, the previews of files in S3 are disabled?" - but that's not the case. I still don't have a good idea of how to phrase it, but I'd say something like, "By default, previews only work for individual files, and don't support HTML files with dependencies on other files in the S3 bucket".

Comment (akarve) [inline docs/Catalog/Preview.md:58]:
```suggestion
preview iframes do not have access to external files in S3.
```

Comment (akarve) [reply docs/Catalog/Preview.md]: OK I now say "external files in S3" and hopefully that clarifies.

File: docs/Catalog/Preview.md [hunk 2]
```diff
@@ -54,16 +54,15 @@ currently supported.
...
-File Server). It is at your discretion which JS library (or libraries)
-you wish to import in your HTML file.
-
-To enable "permissive" visualizations, check the `Enable permissive HTML
-rendering` checkbox in [Bucket settings](Admin.md#buckets). Please note
-that rendering Javascript-enabled HTML files in your S3 bucket or Quilt package
-creates a short-lived **publicly available** session for the lifetime of the
-session, although all session traffic _remains encrypted_.
+The Quilt Catalog supports HTML and JavaScript in preview via iframes. By default,
+preview iframes do not have access to external files in S3.
```
Comment (akarve) [inline docs/Catalog/Preview.md:58]:
```suggestion
preview iframes do not have IAM permissions and are therefore unable to access private files in S3.
```

**Review by dimaryaz — DISMISSED** [review]

File: docs/Catalog/Preview.md [hunk 3]
```diff
@@ -54,16 +54,15 @@ currently supported.
...
-File Server). It is at your discretion which JS library (or libraries)
-you wish to import in your HTML file.
-
-To enable "permissive" visualizations, check the `Enable permissive HTML
-rendering` checkbox in [Bucket settings](Admin.md#buckets). Please note
-that rendering Javascript-enabled HTML files in your S3 bucket or Quilt package
-creates a short-lived **publicly available** session for the lifetime of the
-session, although all session traffic _remains encrypted_.
+The Quilt Catalog supports HTML and JavaScript in preview via iframes. By default,
+preview iframes do not have IAM permissions and are therefore unable to access private files in S3.
```
Comment (akarve) [inline docs/Catalog/Preview.md:58]:
```suggestion
preview iframes do not have IAM permissions and are therefore unable to access
private files in S3.
```

**Review by dimaryaz — APPROVED** [review]

**Pull request merged**